src/lib/stitching/gravity/stitching.ts

```diff
@@ -74,6 +81,35 @@ export const gravityStitchingEnvironment = (
             })
           },
         },
+        formattedEndAt: {
+          fragment: gql`
+            ... on ViewingRoom {
+              startAt
+              endAt
+              timeZone
+            }
+          `,
+          resolve: ({ startAt: _startAt, endAt: _endAt, timeZone }) => {
+            const startAt = moment(_startAt)
+            const endAt = moment(_endAt)
+
+            timeZone = timeZone || moment.tz.guess()
+
+            if (endAt > moment().add(30, "days")) {
+              return null
+            }
+            if (endAt < startAt) {
```

Comment:
Probably `if (now > endAt)` ?

And do we need to worry about `if (now < startAt)`?